### PR TITLE
Add parser for cbt

### PR DESCRIPTION
**Pull request by graipher**
`cbt` is a CLI tool for interacting with Google Cloud Big Table: https://cloud.google.com/bigtable/docs/cbt-overview

It has a rather weird output layout and no json output option.

### CODE REVIEW

File: jc/parsers/cbt.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    $ jc cbt
+
+Usage (module):
+
+    import jc
+    result = jc.parse('cbt', cbt_command_output)
+
+Schema:
+
+    [
```
Comment (kellyjonbrazil) [inline jc/parsers/cbt.py:27]: Maybe have two schemas - one for default and one for raw?

File: jc/parsers/cbt.py [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+            for field in line.splitlines():
+                if not field.strip():
+                    continue
+                if field.startswith(" " * 4):
+                    value = field.strip(' "')
+                    if value_next:
+                        cells.append({
+                            "column_family": column_name.split(":", 1)[0],
+                            "column": column_name.split(":", 1)[1],
+                            "timestamp": datetime.datetime.strptime(timestamp, "%Y/%m/%d-%H:%M:%S.%f").isoformat(),
```
[the remark this answers is not in the document]
Comment (kellyjonbrazil) [reply jc/parsers/cbt.py:164]: I  can add a new timestamp format to `utils.timestamp()` after merging.

**Comment by kellyjonbrazil** [comment]
Nice - thank you for the contribution!

Do you think the datasets could get large enough that it would make sense to make a streaming version of this parser? If so, I can take what you did here and make a new streaming version unless you want to do that.

**Comment by graipher** [comment]
> Nice - thank you for the contribution!
> 
> Do you think the datasets could get large enough that it would make sense to make a streaming version of this parser? If so, I can take what you did here and make a new streaming version unless you want to do that.

A streaming version would definitely be a good option! While doing full table scans or reading keys that are far away from each other (lexicographically) is a thing to be avoided when looking for BigTable performance, reading contiguous key ranges could be a use case for that (as well as the occasional full table scan that cannot be avoided).

File: jc/parsers/cbt.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (graipher) [reply jc/parsers/cbt.py:180]: Note that the string shown seems to depend on the users locale: https://stackoverflow.com/a/69169225/4042267

[the remark this answers is not in the document]
Comment (kellyjonbrazil) [reply jc/parsers/cbt.py:180]: Cool - would you like me to keep the python ISO conversion? I normally do epoch timestamps, but can keep this in there, too. I could also add an ISO string output to the `utils.timestamp()` class as an attribute, like `ts.iso` or something. That might be cleaner.

[the remark this answers is not in the document]
Comment (graipher) [reply jc/parsers/cbt.py:180]: Yeah, an additional attribute would be a good way to go IMO. A plain epoch timestamp is nice for machines, but when a human inspects the JSON an isoformat string is nicer (and still somewhat machine readable at the same time).

**Comment by kellyjonbrazil** [comment]
I merged into `dev` and made some minor changes. Tests are passing. Let me know if you run into any issues. Thanks again!

https://github.com/kellyjonbrazil/jc/blob/dev/jc/parsers/cbt.py

**Pull request merged**